newton/_src/sim/model.py

```
@@ -815,3 +845,28 @@ def get_attribute_frequency(self, name: str) -> ModelAttributeFrequency:
         if frequency is None:
             raise AttributeError(f"Attribute frequency of '{name}' is not known")
         return frequency
+
+    def get_requested_state_attributes(self) -> List[str]:
```

Ruff (0.14.8)

Undefined name `List`

(F821)